src/index.js

```diff
@@ -0,0 +1,65 @@
+'use strict'
+
+const PeerInfo = require('peer-info')
+const Libp2p = require('libp2p')
+const Websockets = require('libp2p-websockets')
+const Secio = require('libp2p-secio')
+const Mplex = require('libp2p-mplex')
+const GossipSub = require('libp2p-gossipsub')
+const PubsubPeerDiscovery = require('libp2p-pubsub-peer-discovery')
+
+/**
+ *
+ * @param {object} [options]
+ * @param {PeerId} [options.peerId]
+ */
+async function create (options = {}) {
+  const peerInfo = await PeerInfo.create(options.peerId)
+
+  // TODO: Reusable peer id
+  const libp2p = await Libp2p.create({
+    peerInfo,
+    modules: {
+      transport: [Websockets],
+      streamMuxer: [Mplex],
+      connEncryption: [Secio],
+      pubsub: GossipSub,
+      peerDiscovery: [PubsubPeerDiscovery]
+    },
+    config: {
+      relay: {
+        enabled: true,
+        hop: {
+          enabled: true,
+          active: false
```

Comment:
Interesting to consider: Using active hopping and connections between relays to enable inter-relay connectivity, so if A dials B over R1 and B isn't connected to R1, R1 (that is connected to R2) will ask it for a connection to B, that way relaying becomes easily scalable